server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts

```diff
@@ -19,106 +20,102 @@ import {
 } from '../../DBService/ClinicalDetails/Mutation';
 import { GetInvestigatedPatientClinicalDetailsFields } from '../../Models/ClinicalDetails/GetInvestigatedPatientClinicalDetailsFields';
 
-const complexityCalculationMessage = 'patient clinical details by status';
 const clinicalDetailsRoute = Router();
-const errorStatusCode = 500;
 
 clinicalDetailsRoute.get('/symptoms', (request: Request, response: Response) => {
     const symptomsLogger = logger.setup({
-        workflow: 'Fetching Symptoms',
+        workflow: 'query all symptoms',
         investigation: response.locals.epidemiologynumber,
         user: response.locals.user.id
     });
-    symptomsLogger.info('launching DB request', Severity.LOW);
-    graphqlRequest(GET_SYMPTOMS, response.locals).then((result: any) => {
-        symptomsLogger.info('got respond from DB', Severity.LOW);
-        response.send(result)
+
+    symptomsLogger.info(launchingDBRequestLog(), Severity.LOW);
+    graphqlRequest(GET_SYMPTOMS, response.locals)
+    .then(result => {
+        symptomsLogger.info(validDBResponseLog, Severity.LOW);
+        response.send(result);
     }).catch(error => {
-        symptomsLogger.error(`got error when approaching the graphql API: ${error}`, Severity.HIGH);
-        response.sendStatus(errorStatusCode);
+        symptomsLogger.error(invalidDBResponseLog(error), Severity.HIGH);
+        response.sendStatus(error);
```

Comment:
Isn't error an object? maybe send instead of sendStatus?

Reply:
same for lines 57, 75, 114

Reply:
error can be either a string or an object. Due to the fact we do want to log it on the client as well, we must send the error as provided because we dont know where the description is

Reply:
yea but why sendStatus instead of just send?